### PR TITLE
Add OWNERS file

### PR DESCRIPTION
**Pull request by prydonius**
cc @migmartri @Angelmmiguel @jackfrancis @technosophos 

@technosophos I've put you down as an assignee since you've helped review PRs, but let me know if you're prefer not be listed here.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+assignees:
```
Comment (Angelmmiguel) [inline OWNERS]: What's the difference between `assignees` and `maintainers`?

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- migmartri
```
Comment (Angelmmiguel) [inline OWNERS]: In the Kubernetes repo, the [OWNERS file](https://github.com/kubernetes/charts/blob/master/OWNERS) has a different indentation. I'm not sure if this kind of files are standardized

File: OWNERS (hunk 1 above)
Comment (prydonius) [inline OWNERS]: This seems to be inconsistent everywhere (core uses approvers and reviewers)... I'm following the example in the template linked from the incubation doc: https://github.com/kubernetes/kubernetes-template-project/blob/master/OWNERS

File: OWNERS (hunk 2 above)
Comment (prydonius) [inline OWNERS]: It's YAML so I believe it doesn't matter, but I'll change it to match the other repos :+1:

File: OWNERS (hunk 1 above)
Comment (Angelmmiguel) [inline OWNERS]: :+1:

**Pull request merged**